blueprints/vendor-shim/files/vendor/__name__-shim.js

```diff
@@ -0,0 +1,9 @@
+(function() {
```

Comment:
Seems fine to me. The `vendor/some-file.js` just triggered flash backs for me :stuck_out_tongue_closed_eyes: